client/src/pages/Contact_form.tsx

```diff
@@ -0,0 +1,129 @@
+import { useState, FormEvent } from 'react';
+import ReCAPTCHA from 'react-google-recaptcha';
+import useRecaptcha from '../../useRecaptcha';
+import styles from './Contact_form.module.css'
+
+export const ContactForm = () => {
+  const [name, setName] = useState('');
+  const [surname, setSurName] = useState('');
+  const [email, setEmail] = useState('');
+  const [message, setMessage] = useState('');
+  const [errors, setErrors] = useState<{ [key: string]: string }>({});
+  const [successMessage, setSuccessMessage] = useState('');
+  const { capchaToken, recaptchaRef, handleRecaptcha } = useRecaptcha();
+
+  const handleSubmit = async (e: FormEvent) => {
+    e.preventDefault();
+    setErrors({});
+    setSuccessMessage('');
+
+    const validationErrors: { [key: string]: string } = {};
+
+    if (!name.trim()) validationErrors.name = "Name is required";
+    if (!surname.trim()) validationErrors.surname = "Surname is required";
+    if (!email.includes('@')) validationErrors.email = "Invalid email";
```

Comment:
use email pattern for this case and implement another check to see if email is empty as well